Sources/Quick/DSL/World+DSL.swift

```diff
@@ -85,6 +85,7 @@ extension World {
     }
 #endif
 
+    @nonobjc
```

Comment:
Is this necessary? Does `World` use `@objcMembers`?

Reply:
Yes it's needed. Otherwise those conflict with `objc_it` and so on.